workshop/content/flink-on-kda/local-flink-development/anatomy-flink-program.en.md

```diff
@@ -4,15 +4,15 @@ chapter: false
 weight: 40
 ---
 
-According to the [Flink documentation](https://ci.apache.org/projects/flink/flink-docs-stable/dev/api_concepts.html#anatomy-of-a-flink-program), a Flink program consists of the following basic parts:
+According to the [Flink documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/overview/#anatomy-of-a-flink-program), a Flink program consists of the following basic parts:
```

Comment:
Can we please pin this to the last stable verison: https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/overview/#anatomy-of-a-flink-program